pgmpy/inference/base_continuous.py

```diff
@@ -0,0 +1,212 @@
+﻿#!/usr/bin/env python3
```

Comment:
@ankurankan I have tried to create base classes. Please check if this what you meant.